packages/web-components/package.json

```diff
@@ -40,11 +40,13 @@
     "@department-of-veterans-affairs/formation": "^7.0.1",
     "@stencil/postcss": "^2.0.0",
     "@stencil/react-output-target": "^0.0.9",
+    "@stencil/sass": "^2.0.0",
     "@types/jest": "^26.0.20",
     "@types/puppeteer": "^5.4.3",
     "@types/react-dom": "16.0.9",
     "@typescript-eslint/eslint-plugin": "^5.30.5",
     "@typescript-eslint/parser": "^5.30.5",
+    "@uswds/uswds": "^3.2.0",
```

Comment:
Because of the way things changed along the way for this component, I ended up adding the new NPM packages to this PR  but if we feel like this and the related files (stencil config, packag.json, and yarn.lock) should be its own PR, I can pull it out.

Reply:
That's fine with me